examples/bridge-app/linux/main.cpp

```diff
@@ -478,6 +549,44 @@ EmberAfStatus HandleReadSwitchAttribute(DeviceSwitch * dev, chip::AttributeId at
     return EMBER_ZCL_STATUS_SUCCESS;
 }
 
+EmberAfStatus HandleReadPowerSourceAttribute(DevicePowerSource * dev, chip::AttributeId attributeId, uint8_t * buffer,
+                                             uint16_t maxReadLength)
+{
+    if ((attributeId == ZCL_POWER_SOURCE_BAT_CHARGE_LEVEL_ATTRIBUTE_ID) && (maxReadLength == 1))
```

Comment:
Please use the namespaced attribute names.

Reply:
Done.